frontend/public/components/notification-drawer.tsx

```diff
@@ -193,63 +193,73 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
 }) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
+  const [rulesAccess, isRulesAccessLoading] = useAccessReview2({
+    group: 'monitoring.coreos.com',
+    resource: 'prometheusrules',
+    verb: 'list',
+  });
   React.useEffect(() => {
-    const poll: NotificationPoll = (url, key: 'notificationAlerts' | 'silences', dataHandler) => {
-      dispatch(alertingLoading(key));
-      const notificationPoller = (): void => {
-        coFetchJSON(url)
-          .then((response) => dataHandler(response))
-          .then((data) => dispatch(alertingLoaded(key, data)))
-          .catch((e) => dispatch(alertingErrored(key, e)))
-          .then(() => (pollerTimeouts[key] = setTimeout(notificationPoller, 15 * 1000)));
+    if (rulesAccess && isRulesAccessLoading) {
```

Comment:
Dont think this is actually right. Shouldn't the `isRulesAccessLoading` be actually set to `false` to proceed ?

Reply:
@jhadvig removed the isloading flag because it wasn't necessary for the check.